data/human/transport missions.txt

```diff
@@ -3209,3 +3209,76 @@ mission "Earth Retirement"
 			label reject
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+
+
+mission "Spacediving professionals"
+	name "Spacediving at <destination>"
+	description "Drop skydivers from the edge of space. They are carrying cooled spacesuits, durable parachutes and oxygen bottles to ensure the survival of their party. Payment is <payment>."
+	passengers 5 10
+	repeat
+	job
+	source
+		government "Republic" "Free Worlds" "Syndicate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
+	to offer
+		and
+			credits >= 1000000
+			random > 96
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		payment 8000
+		dialog "The skydivers jump from your ship and make a spectacular dive until they open their parachutes and land safely near the spaceport. You received payment of <payment> just before they jumped out."
+
+
+
+mission "Spacediving amateurs"
+	name "Spacediving at <destination>"
+	description "Drop skydivers from the edge of space. They are carrying cool wingsuits, fashionable parachutes and beer bottles to ensure the trip will be a great party. Payment is 8000 credits."
+	passengers 5 10
+	repeat 3
+	job
+	source
+		government "Republic" "Free Worlds" "Syndicate"
+	destination
+		distance 7 9
+		government "Republic" "Free Worlds" "Syndicate"
+	to offer
+		and
+			"Spacediving professionals: done" >= 1
+			"Spacediving amateurs: offered" <= 6
+			"Spacediving fools: offered" <= 2
+			credits >= 1000000
+			random > 40
+	on visit
+		dialog phrase "generic passenger on visit"
+	on complete
+		dialog "The skydivers were too affraid to jump and refuse to pay after you land with them. Maybe next time check if your passengers have suitable preparation for a high-altitude jump like this."
```

Comment:
Considering that you've brought them a considerable chunk away from their departing planet, I'd expect they'd at least pay compensation.
```suggestion
		dialog "As you descend through the upper atmosphere, you inform your skydivers that it's time to jump. However, after being able to see the distances involved, all of them are far less keen on the idea than when you first picked them up. You bring them down to the landing pad, and they pay you a meagre <payment> before hopping off."
```

Reply:
They now pay 3000 credits.